timelock-impl/src/main/java/com/palantir/atlasdb/timelock/lock/AsyncLockService.java

```diff
@@ -177,9 +179,11 @@ private AsyncResult<HeldLocks> acquireImmutableTimestampLock(UUID requestId, lon
     }
 
     private AsyncResult<HeldLocks> acquireNamedTimestampLockInternal(
```

Comment:
acquireNamedTimestampLocksInternal

In general, let's make sure you do a naming-alignment pass once we're done with all of the changes.